examples/example_plugin/example_plugin/jobs.py

```diff
@@ -1,5 +1,7 @@
+import sys
 import time
 
+from celery.utils.log import get_task_logger
```

Comment:
It should already be possible to import from there since we have `from celery.utils.log import get_task_logger` in `nautobot.extras.jobs.py`? We'd just need to document it?